Multisig/Data/Services/Safe Client Gateway Service/CollectiblesRequest.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  CollectiblesRequest.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 09.07.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+struct CollectiblesRequest: JSONRequest {
+    let address: String
+
+    var httpMethod: String { return "GET" }
+    var urlPath: String { return "/v1/safes/\(address)/collectibles/" }
+
+    typealias Response = [Collectible]
+    typealias ResponseType = Response
+
+    init(address: Address) {
```

Comment:
When structs have initializers in their struct definitions, Swift does not generate a default initializer. I would like to have that still. Can we move this initializer to the extension on the `CollectiblesRequest`?